Library/Homebrew/test/test_dependency_collector.rb

```diff
@@ -76,6 +76,20 @@ def test_x11_min_version_and_tag
     assert_predicate dep, :optional?
   end
 
+  def test_ant_dep_mavericks_or_newer
+    skip "Only for Mac OS" unless OS.mac?
```

Comment:
And now I see https://github.com/Homebrew/brew/pull/543, which pushes similar tests out of here. Perhaps it would be wisest to remove these two tests altogether...